backend/src/controllers/posts.js

```diff
@@ -1,37 +1,41 @@
 const Post = require('../models/Post')
 
-async function findAll (ctx) {
-  // Fetch all Post’s from the database and return as payload
+const findAll = async (ctx) => {
+  console.log('--------------->  Lista los post!', ctx)
   const posts = await Post.find({})
-  // ctx.body = posts
-  ctx.body = { message: 'greetings from you local API :P' }
+  ctx.body = posts
 }
 
-async function create (ctx) {
+const create = async (ctx) => {
   // Create New Post from payload sent and save to database
-  // const newPost = new Post(ctx.request.body)
-  // const savedPost = await newPost.save()
-  ctx.body = { message: 'greetings from you local API :P' }
+  try {
+    //   console.log('---------------> request body', ctx)
+    const newPost = await new Post(ctx.request.body)
+    //  console.log('---------------> saved post', ctx)
+    await newPost.save()
+    // console.log('---------------> post to json!', ctx)
+    const postedJSON = newPost.toJSON()
+    console.log(postedJSON)
+    ctx.body = { postedJSON }
+  } catch (err) {
+    ctx.throw(409)
+  }
 }
 
-async function destroy (ctx) {
-  // Get id from url parameters and find Post in database
+const destroy = async (ctx) => {
   const id = ctx.params.id
   const post = await Post.findById(id)
```

Comment:
Instead of finding and then removing the Post, you should use mongoose method [findByIdAndRemove](https://mongoosejs.com/docs/api.html#model_Model.findByIdAndRemove)
`await Post.findByIdAndRemove(id)` with no body on the response, just a status code 204